content/src/main/content/jcr_root/apps/acs-commons/config.author/com.adobe.acs.commons.adobeio.core.service.impl.IntegrationServiceImpl.config

```diff
@@ -0,0 +1,15 @@
+getEndpoint="https://ims-na1.adobelogin.com/ims/exchange/jwt"
```

Comment:
Was this committed unintentionally? It seems unlikely we would want to store a private key in this git repository. You will probably need to work with GitHub to expunge this commit from their servers.

Reply:
The information provided in this file is sample. It is not an existing key. But I commit this by mistake. I have to delete those config files.